app/controllers/admin/notes_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Admin
+  class NotesController < ApplicationController
+    def create
+      @submission = Submission.find(note_params[:submission_id]) # require 'pry'; binding.pry
+      note = Note.create(gravity_user_id: @current_user, **note_params) # This doesn't actually seem to render in our layout.
```

Comment:
What do you mean it doesn't render?

Reply:
ugh, these comments seem to have been formatted like so for prettier and i missed them. the 'doesnt seem to render' one is wrong, i needed to use `flash[:notice]` - `[:error]` was not rendering.